test/solidity/Facets/AllBridgeFacet.t.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.17;
+
+import { LibAllowList, TestBaseFacet, console, ERC20 } from "../utils/TestBaseFacet.sol";
+import { AllBridgeFacet, IAllBridge, MessengerProtocol } from "lifi/Facets/AllBridgeFacet.sol";
+import { TestBase } from "../utils/TestBase.sol";
+
+// Stub AllBridgeFacet Contract
+contract TestAllBridgeFacet is AllBridgeFacet {
+    constructor(IAllBridge _allBridge) AllBridgeFacet(_allBridge) {}
+
+    function addDex(address _dex) external {
+        LibAllowList.addAllowedContract(_dex);
+    }
+
+    function setFunctionApprovalBySignature(bytes4 _signature) external {
+        LibAllowList.addAllowedSelector(_signature);
+    }
+}
+
+contract AllBridgeFacetTest is TestBaseFacet {
+    // These values are for Optimism_Kovan
```

Comment:
is this comment accurate? Seems like they are mainnet addresses

Reply:
no they are not, just sloppy copy paste